accepted/010-database.md

```diff
@@ -0,0 +1,41 @@
+# SQLite for Auraed
+
+This feature will accomplish **data persistency after reboots/restarts/failure** because **auraed is a systems daemon built for humans who often make mistakes**.
+
+### Outcomes
+
+  - Allow **persistent records** to exist after a daemon crash, failure.
+  - Allow **persistent records** to exist atomicaly after a gRPC request return. 
+      - There will be a deliberate coupling between the gRPC server and the data layer/model.
+  - Allow **encryption at rest** which can be unlocked using the `/etc/aurae/pki/server.key` (or a cryptographic hash) server key for the database.
+  - Allow **joining** and **connecting** records and schemas. For example we should have a collection of `VirtualMachine` records, which need to be mapped to `NIC` records.
+  - Allow **for a single database file** which can easily be backed up, recovered, and snapshotted for disaster recovery.
```

Comment:
whats the max file size we expect here? can it become a performance factor fo the auraeD startup time?